library/src/main/java/com/owncloud/android/lib/resources/files/RemoveFileRemoteOperation.java

```diff
@@ -35,6 +35,12 @@ public class RemoveFileRemoteOperation extends RemoteOperation {
      */
     public RemoveFileRemoteOperation(String remotePath) {
         mRemotePath = remotePath;
+        sessionTimeOut = SessionTimeOutKt.getDefaultSessionTimeOut();
```

Comment:
```suggestion
        sessionTimeOut = new SessionTimeOut(30000, 5000);
```

Regarding all of the non-default `30000` values: have we considered changing all of them to `40000`? Would this break anything?

Otherwise I would recommend leaving the timeouts as they are.